src/parsing/css-url-image-value-parsing.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//     You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//     See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(internal) {
+
+  function consumeURLImageValue(string) {
+    var parsing = internal.parsing;
+
+    var result = parsing.consumeList([
+      parsing.ignore(parsing.consumeToken.bind(null, /^url\("/i)),
+      parsing.consumeToken.bind(null, /[^"]*/),
+      parsing.ignore(parsing.consumeToken.bind(null, /^"\)/))
+    ], string);
+    if (!result) {
+      return null;
+    }
+    return [[new CSSURLImageValue(result[0][0])], result[1]];
```

Comment:
It still shouldn't be inside a second array though? Just [ parsedTing, remaining ]. If something else needs this to be in an array it should do it there.